app/Http/Controllers/Accessories/AccessoriesController.php

```diff
@@ -127,6 +128,17 @@ public function update(ImageUploadRequest $request, $accessoryId = null)
             return redirect()->route('accessories.index')->with('error', trans('admin/accessories/message.does_not_exist'));
         }
 
+        $min = $accessory->numCHeckedOut();
```

Comment:
Is this a typo? I'm seeing the two capitalizations `numCHeckedOut()` and `numCheckedOut()` in this PR, same as develop.